pkg/logs/client/destination.go

```diff
@@ -7,6 +7,14 @@ package client
 
 import (
 	"net"
+	"sync"
+
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+const (
+	chanSize      = 100
```

Comment:
you should use `config.ChanSize` instead

Reply:
No because of circular import (but this probably fixed when we refactor the packages)

Reply:
👍